gfx/MShockXotto+/pngs_large_64x80/monsters/rid_mon_horse_police.json

```
@@ -0,0 +1,4 @@
+{
+  "id": "rid_mon_horse_police",
```

```suggestion
  "id": [ "rid_mon_horse", "rid_mon_horse_police" ],
```
Otherwise it will stop using the sprite for the regular horse (unless there's another json somewhere setting the regular horse sprite)

---

Huh so it does nice catch ^-^